src/rest_api.py

```diff
@@ -98,6 +100,17 @@ def sync_data():
         logger.info("Calling latest version sync with 'all'")
         _session.post(url, json=['all'])
         resp['message'] = resp['message'] + " for latest version"
+
+    cve_source_sync = input_json.get('cve_source_sync', False)
+    if cve_source_sync:
+        url = "http://{host}:{port}/{endpoint}".format(host=_SERVICE_HOST,
+                                                       port=_SERVICE_PORT,
+                                                       endpoint=_CVE_SOURCE_SYNC_ENDPOINT)
+        logger.info("Calling latest cve source sync")
+        cve_source_sync['ecosystems'] = input_json.get('cve_ecosystem', [])
```

Comment:
what happens if we send cve_source_sync without ecosystem?

Reply:
Well, Since the default value is [] ( empty array), it will not do any modification in the CVE node as it is filtering as per the ecosystems. It is just checks and come out of the job without doing anything. 
Reference: https://github.com/fabric8-analytics/fabric8-analytics-data-model/blob/master/src/utils.py#L274